src/components/ReportActionItem/ReportActionItemImage.tsx

```diff
@@ -109,9 +109,11 @@ function ReportActionItemImage({thumbnail, image, enablePreviewModal = false, tr
                 {({report}) => (
                     <PressableWithoutFocus
                         style={[styles.w100, styles.h100, styles.noOutline as ViewStyle]}
-                        onPress={() => Navigation.navigate(ROUTES.TRANSACTION_RECEIPT.getRoute(report?.reportID ?? '', transaction?.transactionID ?? ''))}
```

Comment:
Looks like receipt modal is already using separate route?

Reply:
Oh same work was being done at the same time in other place. We can close this PR